yt/visualization/volume_rendering/tests/test_scene.py

```diff
@@ -97,7 +98,9 @@ def test_annotations():
         assert np.where((im == c).all(axis=-1))[0].shape[0] > 0
     sc[0].tfh.tf.add_layers(10, colormap="cubehelix")
     sc.save_annotated(
-        "test_scene_annotated.png", text_annotate=[[(0.1, 1.05), "test_string"]]
+        "test_scene_annotated.png",
+        text_annotate=[[(0.1, 1.05), "test_string"]],
+        render=False,
```

Comment:
Indeed the test shouldn't break because of this, but indeed it's not ideal. I don't remember why I set `render=False` here. I was probably just trying to get rid of the warning...